src/HospitalModel.H

```diff
@@ -132,76 +130,38 @@ void HospitalModel<PCType, PTDType, PType>::treatAgents(PCType& a_agents, /*!< A
                                 [=] AMREX_GPU_DEVICE (int i, RandomEngine const& engine)
                                 noexcept
                 {
-                    if ( !isHospitalized(i, ptd) )  {
-                        // agent is not in hospital
-                        return;
-                    }
-                    if (counter_ptrs[d][i] == Math::ceil(latent_per_ptrs[d][i])) {
-                        // agent just started treatment
-                        return;
-                    }
-                    if ( timer_ptrs[d][i] == 0) {
-                        // agent has recovered/died from disease d
-                        return;
-                    }
-                    if ( is_alive_ptr[i] == 0) {
-                        // agent is dead
-                        return;
-                    }
+                    // agent is not in hospital
+                    if (!isHospitalized(i, ptd)) return;
+                    // agent just started treatment at start of symptoms appearing
+                    if (counter_ptrs[d][i] == Math::floor(incubation_per_ptrs[d][i])) return;
+                    // agent has recovered/died from disease d
+                    if (timer_ptrs[d][i] == 0) return;
+                    // agent is dead
+                    if (is_alive_ptr[i] == 0) return;
```

Comment:
`{ return; }` instead of `return;`, unless we are not enforcing that AMReX practice in ExaEpi.

Reply:
Not in the style guide.

Reply:
See this: https://github.com/erf-model/ERF/pull/1410#issuecomment-1919248767

Reply:
Ah, ok, I was only aware of that other document, which Andrew pointed me to.